image.go

```diff
@@ -156,8 +160,19 @@ func (i *image) Labels() map[string]string {
 }
 
 func (i *image) RootFS(ctx context.Context) ([]digest.Digest, error) {
+	if i.diffIDs != nil {
+		return i.diffIDs, nil
```

Comment:
Shouldn't this also be covered by the mutex?

Reply:
nvm. coincident with https://github.com/containerd/containerd/pull/8488#discussion_r1188289107 :man_facepalming: